grammars/css.cson

```diff
@@ -20,294 +20,1235 @@
+          '2':
+            'name': 'punctuation.section.function.begin.bracket.round.css'
+        'end': '\\)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.section.function.end.bracket.round.css'
+        'name': 'meta.function.gradient.css'
+        'patterns': [
+          {
+            'match': '(?i)(?<![\\w-])(from|to|at)(?![\\w-])'
+            'name': 'keyword.operator.gradient.css'
+          }
+          {
+            'include': '#property-values'
+          }
+        ]
+      }
+      # Gradients (Deprecated/original Webkit syntax)
+      {
+        'begin': '(?i)(?<![\\w-])(-webkit-gradient)(\\()'
+        'beginCaptures':
+          '1':
+            'name': 'invalid.deprecated.gradient.function.css'
+          '2':
+            'name': 'punctuation.section.function.begin.bracket.round.css'
+        'end': '\\)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.section.function.end.bracket.round.css'
+        'name': 'meta.function.gradient.invalid.deprecated.gradient.css'
+        'patterns': [
+          {
+            'begin': '(?i)(?<![\\w-])(from|to|color-stop)(\\()'
+            'beginCaptures':
+              '1':
+                'name': 'invalid.deprecated.function.css'
+              '2':
+                'name': 'punctuation.section.function.begin.bracket.round.css'
+            'end': '\\)'
+            'endCaptures':
+              '0':
+                'name': 'punctuation.section.function.end.bracket.round.css'
+            'patterns': [
+              {
+                'include': '#property-values'
+              }
+            ]
+          }
+          {
+            'include': '#property-values'
+          }
+        ]
+      }
+      # Misc functions
+      {
+        'begin': '''(?xi) (?<![\\w-])
+          (annotation|attr|blur|brightness|character-variant|contrast|counters?
+          |cross-fade|drop-shadow|element|fit-content|format|grayscale|hue-rotate
+          |image-set|invert|local|minmax|opacity|ornaments|repeat|saturate|sepia
+          |styleset|stylistic|swash|symbols)
+          (\\()
+        '''
+        'beginCaptures':
+          '1':
+            'name': 'support.function.misc.css'
+          '2':
+            'name': 'punctuation.section.function.begin.bracket.round.css'
+        'end': '\\)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.section.function.end.bracket.round.css'
+        'name': 'meta.function.misc.css'
+        'patterns': [
+          {
+            'match': '(?i)(?<=[,\\s"]|\\*/|^)\\d+x(?=[\\s,"\')]|/\\*|$)'
+            'name': 'constant.numeric.other.density.css'
+          }
+          {
+            'include': '#property-values'
+          }
+          {
+            'match': '[^\'"),\\s]+'
+            'name': 'variable.parameter.misc.css'
+          }
+        ]
+      }
+      # Shapes
+      {
+        'begin': '(?i)(?<![\\w-])(circle|ellipse|inset|polygon|rect)(\\()'
```

Comment:
No. I'm well-aware that `rect()` is non-standard, but it (along with `clip`) is still used in a lot of boilerplate code in the wild. Hence the reason for its inclusion.